File: server/models/Game.py
from sqlalchemy.dialects.postgresql import UUID
from sqlalchemy import Column, Integer, String, ForeignKey
from sqlalchemy.orm import relationship, backref
from . import db


class Game(db.Model):
    __tablename__ = 'game_rooms'

    id = Column(Integer, primary_key=True)
    access_code = Column(String(7), nullable=False)
    game_status = Column(String(120), nullable=False)

    players = relationship('Player', back_populates='game')

    def __repr__(self):
        return f'<Game room: id={self.id}>'

File: server/models/Game.ts
import { DataTypes, Sequelize } from "sequelize";

export default function (sequelize: Sequelize) {
  sequelize.define(
    "Game",
    {
      id: {
        type: DataTypes.INTEGER,
        primaryKey: true,
        autoIncrement: true,
      },

      access_code: {
        type: DataTypes.STRING(7),
        allowNull: false,
        unique: true,
      },
      // game_status: waiting/ready/in_progress/finished
      game_status: {
        type: DataTypes.STRING(120),
        allowNull: false,
      },
    }
  );
}
